### DISCUSSION TITLE
Site search

### DISCUSSION THREAD

**Discussion by michelle-rago** [GSA-TTS/federal-website-standards#233] [Draft]
Provide feedback on the draft standard for site search https://standards.digital.gov/standards/site-search-draft/.

**Comment by mark-meyer** [comment]
Hello, I'm so excited to see these standards taking shape! 

The standard for site search seems like it might be problematic for sites that not content-based but rather provide functionality or tools. One example is GSA 889 tool: https://889.smartpay.gsa.gov/. There is no content on this website — it is a tool to look things up. From a user perspective there doesn't seem to be any value in providing a search bar in the upper right corner since there is not really any content to search.

Will the standards be flexible enough to account for the distinction between content-based sites and websites that are more like tools?

**Reply by michelle-rago** [reply]
Thanks for the comment, Mark! We have heard this from other folks too. We need to figure out how to describe exceptions while still being compliant with 21st Century IDEA. We'll keep your example in mind as we do this.

**Reply by michelle-rago** [reply]
We added exemptions to the site search standard draft https://standards.digital.gov/standards/site-search/. Let me know if you have feedback @mark-meyer.

**Reply by mark-meyer** [reply]
That looks great @michelle-rago!

**Comment by ugopal1** [comment]
Please add www.ebuy.gsa.gov: 
There is no value in providing a search function before user login since there is not really any content to search on this website

**Reply by rphillippi** [reply]
Hi, I believe this falls under the exemptions listed here: https://standards.digital.gov/standards/site-search/

**Comment by hatfieldjm4** [comment]
The "Digital products with minimal content that can easily be navigated without a search function" exemption for the [search metric](https://standards.digital.gov/standards/site-search/#:~:text=by%20the%20public-,Exemptions,-This%20standard%20does) is problematic because it is subjective. 

One unpolished suggestion would be: "Digital products with 2 or fewer publicly-facing subpages of content (excepting any policy subpages, like Privacy Policy), and its public content can easily be navigated without a search function."

**Reply by rphillippi** [reply]
We will consider this. Thank you for the feedback.
